plugins/scaffolder/src/plugin.tsx

```diff
@@ -102,6 +104,11 @@ export const scaffolderPlugin = createPlugin({
           identityApi,
         }),
     }),
+    createApiFactory({
+      api: formDecoratorsApiRef,
+      deps: {},
+      factory: () => DefaultScaffolderFormDecoratorsApi.create(),
```

Comment:
Are you thinking we should be shipping any built-in decorators btw? Or in case that might be a security issue or such, having default ones be provided in the new system as extensions that are disabled by default?

Reply:
Potentially, maybe the ones for provider tokens for sure might be a good idea.